ocaml/xcp-rrdd/lib/plugin/xenctrl_lib.mli

```diff
@@ -0,0 +1,17 @@
+(*
+ * Copyright (C) Cloud Software Group
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+val domain_snapshot :
+     Xenctrl.handle
+  -> float * (Xenctrl.domaininfo * string * int) list * string list
```

Comment:
please add a doc-comment explaining what the function does (provide a list of running, non-migrating vms, and paused ones)